src/actions/posts.js

```diff
@@ -808,27 +814,30 @@ export function getNeededAtMentionedUsernames(state, posts) {
 
         let match;
         while ((match = pattern.exec(post.message)) !== null) {
+            // $FlowFixMe, flow does not recognize the check of !== null in the while loop
```

Comment:
We might be able to fix this by changing the loop to use a `while(true)` instead like
```javascript
while (true) {
    const match = pattern.exec(post.message);
    if (!match) {
        break;
    }

    // ...
}
```